src/app/services/resource.service.ts

```diff
@@ -282,12 +286,16 @@ export class ResourceService {
   getExportDataTypes(constraint: Constraint): Observable<ExportDataType[]> {
     switch (this.endpointMode) {
       case EndpointMode.TRANSMART: {
-        return this.transmartResourceService.getExportFileFormats().pipe(
-          switchMap(fileFormatNames => {
-            return this.transmartResourceService.getExportDataFormats(constraint)
-          }, (fileFormatNames, dataFormatNames) => {
-            return TransmartMapper.mapTransmartExportFormats(fileFormatNames, dataFormatNames);
-          }));
+        if (this.transmartResourceService.exportDataView === 'customFormat') {
```

Comment:
Why ask this property from the `transmartResourceService` and not directly from the config? The config actually tells you if you should use the `transmartResourceService` for exports or not.

Reply:
good point!